docs/layers/primitive-polygon-layer.md

```diff
@@ -1,99 +0,0 @@
-<p class="badges">
```

Comment:
Good suggestion. Let me just change the file name to solid-polygon-layer.md and leave it here